webpack/assets/javascripts/react_app/components/AuthSource/Welcome.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+import EmptyState from '../common/EmptyState';
+import { foremanUrl, getManualURL } from '../../common/helpers';
+
+export const WelcomeAuthSource = ({ canCreate }) => {
+  const content = __(
+    `The authentication process currently requires an LDAP provider, such as <em>FreeIPA</em>, <em>OpenLDAP</em> or <em>Microsoft's Active Directory</em>.`
+  );
+  const description = (
+    <>
+      {__(
+        'Foreman can use LDAP based service for user information and authentication.'
+      )}
+      {<div dangerouslySetInnerHTML={{ __html: content }} />}
+      {
+        <a href={getManualURL('4.1.1LDAPAuthentication')}>
+          {__('Learn more about LDAP authentication in the documentation.')}
+        </a>
+      }
```

Comment:
Also here, no need to wrap components in `{}`. @laviro is it possible to enable some eslint rule to identify such cases of unneeded wrapping in jsx?

Reply:
sure, I think we can manage that,
opened an issue for it: https://projects.theforeman.org/issues/32416